tests/helpers/FakeTLProvider.ts

```diff
@@ -183,7 +183,7 @@ export class FakeTLProvider implements TLProvider {
     })
   }
 
-  public async getRelayVersion(): Promise<string> {
+  public async getVersion(): Promise<string> {
```

Comment:
I don't know the reason for the rename, but at least you forgot to adjust the unit test name for this function.

Reply:
There is no unit test for this function, it's just a mock. The name of the function was changed in `Provider.ts` some time ago. The reason I needed to change it here was to interpret `FakeTLProvider` as a `Provider` to use it as a messaging provider for `Payment` in the unit tests.

Reply:
[RelayProvider.test.ts#L69](https://github.com/trustlines-protocol/clientlib/blob/ba66e0971139b367dedd4bf15eea2e5d3636a0c0/tests/unit/RelayProvider.test.ts#L69) is what I was referring to. This wasn't affected by this PR, but is also bases on the `Provider.getVersion()`. So you don't missed it here, but it was missed as I understand it.
After all: a nitpick.